plenum/server/replica_freshness_checker.py

```diff
@@ -23,36 +22,56 @@ def check_freshness(self, ts):
         '''
         Get all ledger IDs for which
           A) not updated for more than Freshness Timeout
-          B) hasn't been checked for more than than Freshness Timeout
+          B) hasn't been checked (this method is called) for more than than Freshness Timeout
```

Comment:
From code it looks more like "hasn't been attempted to update (returned from this method) for more than timeout"

Reply:
Isn't it the same as `hasn't been checked (this method is called) for more than than Freshness Timeout`?